posix.mak

```diff
@@ -29,7 +29,12 @@ INSTALL_DIR=../install
 DOCDIR=doc
 IMPDIR=import
 
-OPTIONAL_PIC:=$(if $(PIC),-fPIC,)
+# -fPIC is enabled by default and can be disabled with PIC=0
+ifneq ($(PIC),0)
+    DEFAULT_PIC:=-fPIC
```

Comment:
That's what @MartinNowak suggested here: https://github.com/dlang/phobos/pull/5586#discussion_r127688645

I really don't care about the name of this parameter ;-)
If necessary we can always change the name, but I highly doubt wants to deactivate `PIC` (e.g. on Phobos PIE is hard-coded by default for the C files)